docs/how-to-guides/integrating-autoware/creating-your-autoware-meta-repository/creating-autoware-meta-repository.md

```diff
@@ -0,0 +1,57 @@
+# Creating Autoware meta-repository
+
+## What is Meta-repository?
+
+A meta-repository is a repository that manages multiple repositories, and Autoware is one of them.
+It serves as a centralized control point for referencing, configuring, and versioning other repositories.
+
+By using Ansible and vcs, you can automatically set up your Autoware.
```

Comment:
The `vcs` should be capital character.

Reply:
fix https://github.com/autowarefoundation/autoware-documentation/pull/414/commits/06a5739468105b4fa14062901a9543758a579f3f